test/extended/builds/forcepull.go

```diff
@@ -76,13 +75,7 @@ var _ = g.Describe("builds: serial: ForcePull from OpenShift induced builds (vs.
 			exutil.ResetImage(resetData)
 		})
 
-		g.JustBeforeEach(func() {
-			g.By("waiting for builder service account")
-			err := exutil.WaitForBuilderAccount(oc.KubeREST().ServiceAccounts(oc.Namespace()))
-			o.Expect(err).NotTo(o.HaveOccurred())
-		})
```

Comment:
It was defined twice in the file.  I just removed one of the duplicate invocations.